pkg/occlient/occlient.go

```diff
@@ -70,6 +70,9 @@ const (
 	supervisordVolumeName = "odo-supervisord-shared-data"
 )
 
+// WaitForPodTimeOut controls how long we should wait for a pod before giving up
+var WaitForPodTimeOut = 60 * time.Second
```

Comment:
I think that it would be better to have this as `const` in block above

Reply:
Fixed